.github/workflows/e2e_core.yml

```
@@ -190,6 +190,7 @@ jobs:
 
     - name: Run e2e tests
       id: tests
+      continue-on-error: true
       run: ninja -C build-e2e check-sycl-e2e
```

This is what we need.
```suggestion
      run: ninja -C build-e2e check-sycl-e2e || true
```